ambry-account/src/main/java/com/github/ambry/account/MySqlAccountService.java

```diff
@@ -53,6 +54,10 @@
 public class MySqlAccountService extends AbstractAccountService {
 
   private static final Logger logger = LoggerFactory.getLogger(MySqlAccountService.class);
+  private static final String SEPARATOR = "_";
+  private static final int cacheInitialCapacity = 100;
```

Comment:
Minor: add comment that these are for recentNotFoundContainersCache, since we now have multiple caches.